ujt/state.py

```diff
@@ -0,0 +1,108 @@
+from typing import Dict, Tuple, cast
+
+from graph_structures_pb2 import Client, Node
+
+from . import compute_status, generate_data, utils
+from .dash_app import cache
+
+
+def clear_cache():
```

Comment:
Moved all the state modifying methods into one file so different parts of the code can access the underlying data through a unified interface.